lib/Table/TableStyle.ts

```diff
@@ -413,21 +433,25 @@ export default class TableStyle {
     }
 
     const startDates = timeColumn.valuesAsJulianDates.values;
+    const finishDates: (JulianDate | null)[] = new Array(
```

Comment:
hmm, what's the difference between this and the old code?

Reply:
Same as above, these two bits of code can be really slow with lots of dates.
Seems to be a bit faster now using `for` loops instead of `forEach` and `map`